manifests/config.pp

```diff
@@ -19,6 +19,14 @@
     managehome => true,
   }
 
+  file {$::caddy::caddy_ssl_dir:
```

Comment:
Hi, please don't add `::` directly after the `$`. This is a deprecated style that was required for Puppet 3.x

Reply:
changed, also for previously defined variables.